### DISCUSSION TITLE
Any guide for building the System.Maui solution

### DISCUSSION THREAD

**Discussion by mano-code** [dotnet/maui#194] [General]
I just cloned the System.Maui solution to get going but hit the blocker with not finding any document to get started. Please advise.

I couldn't either build it in VS 2019 or from command prompt using MSBuild.

**Comment by mrwcjoughin** [comment]
It’s not out yet - they have only just started working on it

If you want to develop now use Xamarin.Forms, it’s the thing MAUI will
effectively be the next version of when it comes out Nov 2021

On Sun, 02 Aug 2020 at 10:09, mano <notifications@github.com> wrote:

> I just cloned the System.Maui solution to get going but hit the blocker
> with not finding any document to get started. Please advise.
>
> I couldn't either build it in VS 2019 or from command prompt using MSBuild.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/maui/discussions/194>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACOGERSKLMK4TJ7QWW7ANQTR6UNLJANCNFSM4PSNTJJQ>
> .
>
-- 
Thanks,

Matthew Joughin
Founder / CTO

+27 81 529 7169
http://www.adaptableapps.net

**Comment by mano-code** [comment]
Sorry for not being clear first. I wanted to contribute to Maui open source project through development. So wanted to setup my development environment first to have a crack. Thanks.

Cheers
Manohar
+64 27 341 7636

________________________________

**Comment by mrwcjoughin** [comment]
Aah ok - well I think they will start committing code soon

On Sun, 02 Aug 2020 at 22:37, mano <notifications@github.com> wrote:

> Sorry for not being clear first. I wanted to contribute to Maui open
> source project through development. So wanted to setup my development
> environment first to have a crack. Thanks.
>
> Cheers
> Manohar
> +64 27 341 7636
>
> ________________________________
> From: Matthew Joughin <notifications@github.com>
> Sent: Sunday, August 2, 2020 11:33:32 PM
> To: dotnet/maui <maui@noreply.github.com>
> Cc: mano <manooh@live.com>; Author <author@noreply.github.com>
> Subject: Re: [dotnet/maui] Any guide for building the System.Maui solution
> (#194)
>
>
> It’s not out yet - they have only just started working on it
>
> If you want to develop now use Xamarin.Forms, it’s the thing MAUI will
> effectively be the next version of when it comes out Nov 2021
>
> On Sun, 02 Aug 2020 at 10:09, mano <notifications@github.com> wrote:
>
> > I just cloned the System.Maui solution to get going but hit the blocker
> > with not finding any document to get started. Please advise.
> >
> > I couldn't either build it in VS 2019 or from command prompt using
> MSBuild.
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/dotnet/maui/discussions/194>, or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ACOGERSKLMK4TJ7QWW7ANQTR6UNLJANCNFSM4PSNTJJQ
> >
> > .
> >
> --
> Thanks,
>
> Matthew Joughin
> Founder / CTO
>
> +27 81 529 7169
> http://www.adaptableapps.net
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub<
> https://github.com/dotnet/maui/discussions/194#discussioncomment-45644>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/ACSNQEWCY3S2ZCMEGAT4XGLR6VFIZANCNFSM4PSNTJJQ
> >.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/maui/discussions/194#discussioncomment-45729>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACOGERQW42KE3H5GZNRH7NLR6XE7PANCNFSM4PSNTJJQ>
> .
>
-- 
Thanks,

Matthew Joughin
Founder / CTO

+27 81 529 7169
http://www.adaptableapps.net
